dev-docs/bidders/adverxo.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: bidder
+title: Adverxo
+description: Prebid Adverxo Bidder Adapter
+pbjs: true
+pbs: true
+pbs_app_supported: true
+biddercode: adverxo
+userIds: 
+media_types: banner, native, video
+schain_supported: true
+dchain_supported: false
+fpd_supported: true
```

Comment:
You cannot claim to support FPD without describing specifically how you support it. e.g. site.ext.data, user.ext.data, site.content.data, user.data, imp.ext.data

Reply:
Done.